addons/stock/static/src/components/reception_report_main/stock_reception_report_main.js

```diff
@@ -15,30 +15,47 @@ export class ReceptionReportMain extends Component {
         };
         this.ormService = useService("orm");
         this.actionService = useService("action");
+        this.routerService = useService("router");
         this.reportName = "stock.report_reception";
-        const defaultDocIds = Object.entries(this.context).find(([k,v]) => k.startsWith("default_"));
-        this.contextDefaultDoc = { field: defaultDocIds[0], ids: defaultDocIds[1] };
         this.state = useState({
             sourcesToLines: {},
         });
         useBus(this.env.bus, "update-assign-state", (ev) => this._changeAssignedState(ev.detail));
 
         onWillStart(async () => {
+            // Check the router if report was already loaded.
+            let defaultDocIds;
+            const { rfield, rids } = this.routerService.current.hash;
+            if (rfield && rids) {
+                const parsedIds = JSON.parse(rids);
+                defaultDocIds = [ rfield, parsedIds instanceof Array ? parsedIds : [parsedIds] ];
+            } else {
+                defaultDocIds = Object.entries(this.context).find(([k, v]) => k.startsWith("default_"));
```

Comment:
Shouldn't we take `'default_picking_id'` and `active_id` as fallback for `defaultDocIds` ?
Because writing the ids in the URL seems redundant since we already have the `active_id`.
For the model, the traceback is still here if we open the report for a record we aren't supposed to (so, no `data.doc`) and then refresh the page.
If we use `default_picking_id` as default value, we will avoid the error for sure